lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -607,6 +608,28 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  // https://dom.spec.whatwg.org/#dom-document-createcdatasection
+  createCDATASection(data) {
+    if (data.indexOf("[[>") !== -1) {
```

Comment:
Sorry I haven't had time to do a full review of this, but since this subthread seems to be getting a bit out of hand: just use `data.includes("]]>")` instead of any indexOf tricks.